src/Monster.h

```diff
@@ -165,8 +165,8 @@ class CMonster
 	void PreDraw();
 	void Draw();
 	void PostDraw();
-protected:
     void Init(CMonsterDef *pmd);
+protected:
```

Comment:
can you leave the label in-place, but change it to public? maybe line 168 should be `public: //protected:` so that it's obvious which methods had visibility changed for testing purposes. 

I wonder if there is a way to make full inheritance protection work with unit tests. Seems tricky. 🤔